modules/monitoring-accessing-third-party-monitoring-uis.adoc

```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// * monitoring/accessing-third-party-monitoring-uis-and-apis.adoc
+
+:_content-type: CONCEPT
+[id="accessing-third-party-monitoring-uis"]
+= Accessing third-party monitoring UIs
+
+[role="_abstract"]
+{product-title} does not provide or support direct access to third-party web user interfaces (UIs) for the following components in the monitoring stack: Alertmanager, Thanos Ruler, and Thanos Querier. 
+As an alternative to these third-party UIs, navigate to the *Observe* section of the {product-title} web console to access metrics, alerting, metrics targets, and dashboard UIs for platform components.
+
+[NOTE]
+====
+Although you can access the third-party Grafana and Prometheus web UIs from the web console or the CLI, this access is deprecated and is planned to be removed in a future {product-title} release.
```

Comment:
can we say that it's going to be removed in the next release (instead of "a future release)?

Reply:
> can we say that it's going to be removed in the next release (instead of "a future release)?

@simonpasquier Are we 100% certain it will be removed in the next release? In general, we try to avoid referring to specific future releases, but in this case it might be ok.

Reply:
Pretty sure it will be. The feedback we had before is that deprecating Grafana without a firm date wasn't nice for users. Maybe express that the UIs could be removed in 4.11 at the earliest?

Reply:
Will change it to "next release"